source/_integrations/proximity.markdown

```diff
@@ -52,6 +52,11 @@ proximity:
     unit_of_measurement: mi
 ```
 
+### Video Tutorial
+This video tutorial explains how to set up geofencing in Home Assistant using the proximity integration.
```

Comment:
```suggestion
This video tutorial explains how to set up geofencing in Home Assistant using the proximity integration. This can be used, for example, to turn down the heat after you've left the house and turn it up again when you are approaching your home.

In the video, you're instructed to restart Home Assistant after changing the configuration file. This is no longer required. You can now just {% term reload %} the file.
```
From just reading _geofencing_ and _proximity_ integration I didn't connect the dots what this could be used for. 
I'm not 100% sure, but I think you can now just reload the file. Or is it different in this case?